File: VB/CustomWizardExample/Wizard/Model/CustomDataSourceType.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.DataAccess.Wizard.Model

Namespace CustomWizardExample.Wizard.Model
	Public Class CustomDataSourceType
		Inherits DataSourceType

		Public Shared ReadOnly Persons As New CustomDataSourceType()
	End Class
End Namespace

File: CS/CustomWizardExample/Wizard/Model/CustomDataSourceType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.DataAccess.Wizard.Model;

namespace CustomWizardExample.Wizard.Model {
    public class CustomDataSourceType : DataSourceType {
        public static readonly CustomDataSourceType Persons = new CustomDataSourceType();
    }
}
